unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs

```diff
@@ -95,6 +101,25 @@ public virtual void SetData(LoadParcelScenesMessage.UnityParcelScene data)
 
             OnSetData?.Invoke(data);
         }
+        
+        internal long EntityFromLegacyEntityString(string entityId)
```

Comment:
This is a pure function, we can make it static (and this also saves the need of having to create a `ParcelScene` component to test this).